lmms_eval/tasks/voicebench/utils.py

```
@@ -0,0 +1,1547 @@
+            return "A"
+        elif 'would be "a.' in response:
+            return "A"
+        elif "the best option from the given choices would be a scorpion (a)" in response:
+            return "A"
+        else:
+            return None
+
+    ground_truth = get_column_value(doc, ["reference"])
+    cnt = 0
+    for idx in range(len(results)):
+        if results[idx] == None:
```

Ruff (0.12.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)